client/config/environment.js

```diff
@@ -65,10 +64,10 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' 'https://www.google.com' 'www.google-analytics.com'",
```

Comment:
I think to get away from the inline script approach we would have to model our solution in a similar fashion to how their are loading the live reload script with ember-cli [ember-cli-inject-live-reload](https://github.com/rwjblue/ember-cli-inject-live-reload).  

I'm thinking we should make create an issue for this and circle back later on this.  Maybe give a PR to [ember-cli-google-analytics] (https://github.com/pgrippi/ember-cli-google-analytics#configuration) at that time and migrate to using that.